test/lib/validation.ts

```diff
@@ -66,9 +68,32 @@ expect.extend({
         const valid = validate(actual.result);
 
         if (!valid) {
+            const extendedError: ExtendetError[] = validate.errors!.map((error): ExtendetError => {
+                return Object.assign(error, { value: _.get(actual.result, error.instancePath.replaceAll("/", ".").slice(1)) });
+            });
             return {
                 pass: false,
-                message: () => `expected a successful result to match the schema '${schemaName}', but got the following errors: ${JSON.stringify(validate.errors, null, 2)}`
+                message: () => `expected a successful result to match the schema '${schemaName}', but got the following errors: ${JSON.stringify(extendedError, null, 2)}`
+            };
+        }
+
+        return { pass: actual.isSuccess, message: () => "" };
+    },
+
+    toBeSuccessfulVoidResult(actual: ConnectorResponse<unknown>) {
+        if (!(actual instanceof ConnectorResponse)) {
+            return { pass: false, message: () => "expected an instance of Result." };
+        }
+
+        if (!actual.isSuccess) {
+            const message = `expected a successful result; got an error result with the error message '${actual.error.message}'.`;
+            return { pass: false, message: () => message };
+        }
+
+        if (typeof actual.result !== "undefined") {
+            return {
+                pass: false,
+                message: () => `expected a successful result to to be a void result, but got the following result: ${JSON.stringify(actual.result, null, 2)}`
```

Comment:
Last comment, rest LGTM.

"Expected a successful result to ~~to~~ be a void result".